### ISSUE TITLE
Cannot create an account with an email

### ISSUE THREAD

**Issue by lucassz** [burtonator/polar-bookshelf#1591]
**Describe the bug**
When trying to create a new account on Mac OS with an email, I receive the verification email but get the error "Cloud function failed: 500:" when I input it into the software.

**Screenshots**
<img width="483" alt="image" src="https://user-images.githubusercontent.com/4793515/107814878-94665100-6d40-11eb-9b1d-443ff988e378.png">

**Desktop (please complete the following information):**
 - OS: Mac OS 10.15.4
 - Version: 2.0.103

**Additional context**
<img width="545" alt="image" src="https://user-images.githubusercontent.com/4793515/107815091-e0b19100-6d40-11eb-86ae-d55ed1e65b39.png">

The following is being returned from that URL:
`"Could not handle HTTP POST request at: /{\n  \"email\": \"[redacted email]\",\n  \"challenge\": \"904606\"\n}"`

**Comment by xdhmoore** [comment]
I am also experiencing this in both the desktop version (also 2.0.103.0), and also in the web version in Firefox.

**Comment by burtonator** [comment]
I'm working on a fix...

**Comment by lucassz** [comment]
Still having this issue, FWIW.

**Comment by BrandonKMLee** [comment]
@lucassz I don't think that this app would even require a need for registration if someone were to use Syncthing or some other to host their own home server. If it is a "personal knowledge repository" then registration should be optional. I am just hoping that there is an alternative to  Liquidtext and Marginote that is made for research and study.
